### DISCUSSION TITLE
schema code for dynamic keys in config.

### DISCUSSION THREAD

**Discussion by SKM162** [carueda/tscfg#220] [Q&A] [answered]
Does the generated classes only hold config values with fixed keys?
or Is it possible to handle cases like:
```javascript
authors: {
    alice {
      email: 0
      country: venice
    }
    bob {
      email: "bob@hotmail.com"
      country: rio
    }
    tim {
      email: "tim@yahoo.com"
      country: czech
    }
  }
```
which is based on the following json schema:
```javascript
authors { 
      type: object  
      additionalProperties {
        type: object
        additionalProperties: false
        properties {
          email {
            type: string
          }
          country {
            type: string
          }
        }
      }
}
``` 

where the author names alice, bob, tim are unknown but the value is an object of known structure. 
so the class must have Map<string, known object> 
what is the tscfg schema dsl code to handle this case?

**Comment by carueda** [comment, marked as answer]
Unfortunately, this is not implemented. Maybe a future version will include this as part of general schema-related adjustments.
